packages/kbn-apm-synthtrace/src/lib/apm/client/apm_synthtrace_es_client/apm_pipeline.ts

```diff
@@ -26,9 +27,13 @@ export function apmPipeline(logger: Logger, version: string, includeSerializatio
       createTransactionMetricsAggregator('1m'),
       createTransactionMetricsAggregator('10m'),
       createTransactionMetricsAggregator('60m'),
-      createServiceMetricsAggregator('1m'),
-      createServiceMetricsAggregator('10m'),
-      createServiceMetricsAggregator('60m'),
+      ...(semver.lt(semver.coerce(version)?.version ?? version, '8.7.0')
+        ? []
+        : [
+            createServiceMetricsAggregator('1m'),
+            createServiceMetricsAggregator('10m'),
+            createServiceMetricsAggregator('60m'),
```

Comment:
These were introduced in 8.7.0. Added this condition to be able to reproduce the same scenario that happens when APM < 8.7.0 ships data.

Reply:
why only service metrics aggregator?

>From 8.7.0 onwards, all aggregated metrics are aggregated at 3 different granularity: 1m, 10m, and 60m.


https://github.com/elastic/apm-server/blob/7b7195e5686eb10e3ffd60b588af9007a3378079/dev_docs/trace_metrics.md#trace-metrics

Reply:
@kpatticha I had to keep transaction 1m and span 1m for < 8.7. The other aggregations were added later https://github.com/elastic/apm-server/issues/9703. But those were always there: https://github.com/elastic/kibana/commit/c224972f470da4a1aad4eac96e61bf3988f0a60f#diff-060ec0857d4febd8ee7f26137ebe6b749ac0313dc9563f651d6d5330ee55258e.